toolchain/sem_ir/formatter.cpp

```diff
@@ -125,23 +231,27 @@ class FormatterImpl {
   }
 
   // Formats a top-level scope, particularly Constants and ImportRefs.
-  auto FormatScope(InstNamer::ScopeId scope_id, llvm::ArrayRef<InstId> block)
-      -> void {
+  auto FormatScope(InstNamer::ScopeId scope_id, llvm::ArrayRef<InstId> block,
+                   bool tentative = false) -> void {
     if (block.empty()) {
       return;
     }
 
     llvm::SaveAndRestore scope(scope_, scope_id);
     out_ << inst_namer_->GetScopeName(scope_id) << " ";
     OpenBrace();
-    FormatCodeBlock(block);
+    FormatCodeBlock(block, tentative);
```

Comment:
This is the only spot that does tentative code block formatting, maybe the branched logic should be inlined here? i.e., per above comment about `tentative=true`, this is the only call to `FormatCodeBlock` with `tentative`, so maybe:

```suggestion
    for (const InstId inst_id : block) {
      TentativeOutputScope scope(*this);
      tentative_inst_chunks_[inst_id.index] = scope.index;
      FormatInst(inst_id);
    }
```

Reply:
Done